src/commands/dev/exec.js

```diff
@@ -24,14 +23,12 @@ class ExecCommand extends Command {
     }
 
     const envSettings = await getEnvSettings(site.root)
-    if (envSettings.file) {
-      console.log(
-        `${NETLIFYDEVLOG} Overriding the following env variables with ${chalk.blue(
-          path.relative(site.root, envSettings.file)
-        )} file:`,
-        chalk.yellow(Object.keys(envSettings.vars))
+    if (envSettings.vars.length > 0) {
```

Comment:
Previous logic had an edge case where it would log if you had a `.env` file with no vars